installer/install.iss.in

```diff
@@ -7,8 +7,9 @@
 #else
 #define APP_VERSION   '%APPVERSION%'
 #endif
-#define APP_URL       'http://msysgit.github.io/'
-#define APP_BUILTINS  'etc\fileList-builtins.txt'
+#define MINGW_BITNESS '%MINGW_BITNESS%'
+#define APP_URL       'http://git-for-windows.github.io/'
+#define APP_BUILTINS  'share\git\builtins.txt'

```

Comment:
Correct. This file will only be present with the next release of the `mingw-w64-git` packages (which takes quite a while now, thanks to the documentation builds).